INTITA/Networking/ApiURL.swift

```diff
@@ -100,6 +108,8 @@ enum ApiURL {
             return "currentUser"
         case .logout:
             return nil
+        case .editUser:
+            return nil
```

Comment:
Please add filename(find the file in chat)

Reply:
Done.